src/index.html

```diff
@@ -1,13 +1,60 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <div class="conteiner">
+
+      <div class="Product">
+
+        <div class="Product__img" href="#">
+          <img src="images/imac.jpeg" 
+               alt="APPLE A1419 iMac 27 "
+               value="APPLE A1419 iMac 27 Retina 5K (MNED2UA)"
+               >
+        </div>
+
+        <div class="Product__name">
+          <h1>
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h1>
+        </div>
+
+        <div class="Product__code">
+          <h2>
+            Product code: 195434
+          </h2>
+        </div>
+
+        <div class="stars">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <h3>
+            Reviews: 5
+          </h3>
+          
+        </div>
+
+        <div class="Product__price">
+          <h4>Price</h4>
+          <span>$ 2,199</span>
+        </div>
+
+        <div class="Product__butoon">
+          <a href="#"
+             class="link"
+             data-qa="hover">Buy
+          </a>
```

Comment:
```suggestion
          <a 
            href="#"
            class="link"
            data-qa="hover"
          >
            Buy
          </a>
```

Reply:
still not resolved
![image](https://github.com/mate-academy/layout_product-cards/assets/55200530/40dd2028-193f-4b79-9991-a35eec9678ac)